macros/SpecName.ejs

```diff
@@ -848,24 +836,24 @@ var specList = {
         url  : 'http://dev.w3.org/2006/webapi/selectors-api2/'
     },
     'Shadow DOM':{
-        name : 'Shadow DOM',
-        url  : 'https://www.w3.org/TR/shadow-dom/'
+        name : 'DOM (Shadow trees)',
```

Comment:
The following pages use the key 'Shadow DOM'. If we update these to 'DOM WHATWG', we could remove the 'Shadow DOM' entry altogether.

https://developer.mozilla.org/en-US/docs/Web/API/Element
https://developer.mozilla.org/en-US/docs/Web/API/Text
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/content
https://developer.mozilla.org/en-US/docs/Web/Web_Components
https://developer.mozilla.org/en-US/docs/Web/Web_Components/Shadow_DOM
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Shadow
https://developer.mozilla.org/en-US/docs/Web/API/Event/Event
https://developer.mozilla.org/en-US/docs/Web/API/HTMLContentElement
https://developer.mozilla.org/en-US/docs/Web/API/HTMLContentElement/select
https://developer.mozilla.org/en-US/docs/Web/API/HTMLContentElement/getDistributedNodes
https://developer.mozilla.org/en-US/docs/Web/API/HTMLShadowElement
https://developer.mozilla.org/en-US/docs/Web/API/HTMLShadowElement/getDistributedNodes
https://developer.mozilla.org/en-US/docs/Web/API/Element/createShadowRoot
https://developer.mozilla.org/en-US/docs/Web/API/Element/shadowRoot
https://developer.mozilla.org/en-US/docs/Web/API/ShadowRoot
https://developer.mozilla.org/en-US/docs/Web/API/Element/attachShadow
https://developer.mozilla.org/en-US/docs/Web/API/ShadowRoot/host
https://developer.mozilla.org/en-US/docs/Web/API/ShadowRoot/innerHTML
https://developer.mozilla.org/en-US/docs/Web/API/HTMLSlotElement
https://developer.mozilla.org/en-US/docs/Web/API/HTMLSlotElement/assignedNodes
https://developer.mozilla.org/en-US/docs/Web/API/HTMLSlotElement/name
https://developer.mozilla.org/en-US/docs/Web/API/Element/assignedSlot
https://developer.mozilla.org/en-US/docs/Web/API/Element/slot
https://developer.mozilla.org/en-US/docs/Web/API/Event/deepPath
https://developer.mozilla.org/en-US/docs/Web/API/DocumentOrShadowRoot
https://developer.mozilla.org/en-US/docs/Web/API/DocumentOrShadowRoot/activeElement
https://developer.mozilla.org/en-US/docs/Web/API/DocumentOrShadowRoot/styleSheets
https://developer.mozilla.org/en-US/docs/Web/API/DocumentOrShadowRoot/getSelection
https://developer.mozilla.org/en-US/docs/Web/API/DocumentOrShadowRoot/elementFromPoint
https://developer.mozilla.org/en-US/docs/Web/API/DocumentOrShadowRoot/elementsFromPoint
https://developer.mozilla.org/en-US/docs/Web/API/DocumentOrShadowRoot/caretPositionFromPoint
https://developer.mozilla.org/en-US/docs/Web/API/Text/assignedSlot